data/mods/Magiclysm/recipes/blacksmithing.json

```diff
@@ -24,5 +24,31 @@
     "book_learn": [ [ "welding_book", 4 ] ],
     "tools": [ [ [ "demon_forge", 15 ] ], [ [ "tongs", -1 ] ] ],
     "components": [ [ [ "scrap_bronze", 9 ] ], [ [ "mercury", 400 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "fireproof_mortar",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_OTHER",
+    "skill_used": "fabrication",
+    "difficulty": 2,
+    "time": "20 m",
+    "book_learn": [ [ "metal_legends", 2 ] ],
+    "qualities": [ { "MANA_INFUSE", "level": 1 } ],
```

Comment:
these are wrong. i don't know the correct thing though. it should be key: value, and the first thing doesn't have both